### PR TITLE
[Bug](decimalv3) fix failed on test_dup_tab_decimalv3 due to wrong precision

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
fix failed on test_dup_tab_decimalv3 due to wrong precision

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 51.59 seconds
 stream load tsv:          508 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17165544555 Bytes

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 53.23 seconds
 stream load tsv:          510 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.4 seconds inserted 10000000 Rows, about 340K ops/s
 storage size: 17169704157 Bytes

File: be/src/vec/exec/scan/vscan_node.cpp [hunk 1]
```diff
@@ -65,9 +65,12 @@
 
 namespace doris::vectorized {
 
-#define RETURN_IF_PUSH_DOWN(stmt)            \
+#define RETURN_IF_PUSH_DOWN(stmt, state)     \
```
Comment (yiguolei) [inline be/src/vec/exec/scan/vscan_node.cpp:68]: not use state, use status please.

File: be/src/vec/exec/scan/vscan_node.cpp [hunk 2]
```diff
@@ -472,34 +475,44 @@ Status VScanNode::_normalize_predicate(const VExprSPtr& conjunct_expr_root, VExp
             }
             if (_is_predicate_acting_on_slot(cur_expr, in_predicate_checker, &slot, &range) ||
                 _is_predicate_acting_on_slot(cur_expr, eq_predicate_checker, &slot, &range)) {
+                Status state = Status::OK();
```
Comment (yiguolei) [inline be/src/vec/exec/scan/vscan_node.cpp:478]: use status not state

File: be/src/vec/exec/scan/vscan_node.cpp [hunk 3]
```diff
@@ -772,6 +785,11 @@ Status VScanNode::_normalize_in_and_eq_predicate(VExpr* expr, VExprContext* expr
                         temp_range, reinterpret_cast<void*>(&val),
                         ColumnValueRange<T>::add_fixed_value_range, fn_name));
             } else {
+                if (sizeof(typename PrimitiveTypeTraits<T>::CppType) != value.size) {
+                    return Status::InternalError(
```
Comment (yiguolei) [inline be/src/vec/exec/scan/vscan_node.cpp:789]: Does this bugfix related with this PR?

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 54.93 seconds
 stream load tsv:          547 seconds loaded 74807831229 Bytes, about 130 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17163996177 Bytes

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Pull request merged**